lib/oli_web/components/delivery/scored_activities/scored_activities.ex

```diff
@@ -0,0 +1,624 @@
+        {#if @current_assessment != nil}
+          <div class="flex flex-col">
+            {#if @current_assessment.container_label}
+              <h4 class="torus-h4 whitespace-nowrap">{@current_assessment.container_label}</h4>
+              <span class="text-lg">{@current_assessment.title}</span>
+            {#else}
+              <h4 class="torus-h4 whitespace-nowrap">{@current_assessment.title}</h4>
+            {/if}
+            <button
+              class="btn btn-primary whitespace-nowrap mr-auto my-6"
+              phx-click="back"
+              phx-target={@myself}
+            >Go back</button>
+          </div>
+        {#else}
+          <h4 class="torus-h4 whitespace-nowrap">Scored Activities</h4>
+        {/if}
+        <div class="flex flex-col">
+          <form
+            for="search"
+            phx-target={@myself}
+            phx-change="search_assessment"
+            class="pb-6 lg:ml-auto lg:pt-7"
+          >
+            <SearchInput.render
+              id="assessments_search_input"
+              name="assessment_name"
+              text={@params.text_search}
+            />
+          </form>
+          {#if @current_assessment != nil}
+            <div id="student_attempts_summary" class="flex flex-row mt-auto">
+              <span class="text-xs">
+                {#if @students_with_attempts_count == 0}
+                  No student has completed any attempts.
+                {#else}
+                  {@students_with_attempts_count} {Gettext.ngettext(OliWeb.Gettext, "student has", "students have", @students_with_attempts_count)} completed {@total_attempts_count} {Gettext.ngettext(OliWeb.Gettext, "attempt", "attempts", @total_attempts_count)}.
+                {/if}
+              </span>
+              {#if @students_with_attempts_count < Enum.count(@students)}
+                <div class="flex flex-col">
+                  <span class="text-xs ml-2">
+                    {Enum.count(@student_emails_without_attempts)} {Gettext.ngettext(
+                      OliWeb.Gettext,
+                      "student has",
+                      "students have",
+                      Enum.count(@student_emails_without_attempts)
+                    )} not completed any attempt.
+                  </span>
+                  <input
+                    type="text"
+                    id="email_inputs"
+                    class="form-control hidden"
+                    value={Enum.join(@student_emails_without_attempts, "; ")}
+                    readonly
+                  />
+                  <button
+                    id="copy_emails_button"
+                    class="text-xs text-primary underline ml-auto mb-6"
+                    phx-hook="CopyListener"
+                    data-clipboard-target="#email_inputs"
+                  ><i class="fa-solid fa-copy mr-2" />{Gettext.ngettext(
+                      OliWeb.Gettext,
+                      "Copy his email address",
```

Comment:
NIT: Could be a "she" and not a "he". Perhaps we can change it to just "Copy email address" and "Copy email addresses"?